.github/workflows/edge.yml

```diff
@@ -4,9 +4,19 @@ on:
   push:
     branches:
       - master
+    paths-ignore:
+      - 'docs/**'
+      - 'docs-web/**'
+      - 'examples/**'
```

Comment:
Might be good to add the following at some point?:

`- 'examples-of-custom-resources/**'`

Reply:
Thanks, I'll create a new PR and include that directory.